features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -585,6 +587,19 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
 
     }, [ initialValues ]);
 
+
+    /**
+     * Check whether which access token type is enabled.
+     */
+    useEffect(() => {
+
+        if (initialValues?.accessToken) {
+            setJWTAccessTokenTypeSelected(initialValues?.accessToken?.type === JWT);
+        } else {
+            setJWTAccessTokenTypeSelected(metadata?.accessTokenType?.defaultValue === JWT);
+        }
+    }, [ initialValues ]);
```

Comment:
Don't we need to add `metadata` as a dependency here?

Also, lets update the pr title to accurately describe the changes in this pr :)